modules/tournament/src/main/PairingRepo.scala

```diff
@@ -240,10 +244,39 @@ final class PairingRepo(coll: Coll)(implicit ec: scala.concurrent.ExecutionConte
         Project(
           $doc(
             "_id" -> false,
-            "w"   -> true,
-            "t"   -> true,
-            "b1"  -> $doc("$cond" -> $arr("$b1", 1, 0)),
-            "b2"  -> $doc("$cond" -> $arr("$b2", 1, 0))
+            //copilot #3
+            "w" -> $doc(
```

Comment:
Suggestion - feels like you could move this logic into the "st" field above and then just use that?